### ISSUE TITLE
mpcorb.n_oppositions is said to contain planet radius, why?

### ISSUE THREAD

**Issue by abitrolly** [esa/pykep#50]
http://esa.github.io/pykep/documentation/planets.html#PyKEP.planet.mpcorb.n_oppositions

```
n_oppositions
    The planet radius Example:
    R = apophis.n_oppositions
```

Why number of oppositions means planet radius?

**Comment by darioizzo** [comment]
probably a typo ...

**Comment by abitrolly** [comment]
Yes. Looks like a mistake.

**Comment by darioizzo** [comment]
I fixed this for the 2.0 release #71
